resources/TrialApp.didact.md

```diff
@@ -1,16 +1,16 @@
 # Welcome to your Trial App Didact Tutorial
 
-This guide will show you how to make and deploy changes to your Azure App Service Trial App. Your trial will last 60 minutes. You can find more information about Azure App Service [here](https://docs.microsoft.com/en-us/azure/app-service/).
+This tutorial will show you how to make and deploy changes to your Azure App Service trial app. Your trial will last 60 minutes. You can find more information about Azure App Service [here](https://docs.microsoft.com/en-us/azure/app-service/).
 
 ## Getting Started
 
 ### Step 1: Clone the trial app source code
 
-1. Locate your imported Trial App in the Azure view.
+1. Locate your imported trial app in the Azure view.
 
 <img src="./trialAppTutorial/viewTrialAppScreenshot.png" style="width: 256px;">
 
-2. Right click on the Trial App.
+2. Right click on the trial app.
```

Comment:
On line 49, you use right-click, but here you use right click. I think it's technically right-click, so I'd just change this one (but more important to just be consistent)